.eslintrc.yaml

```diff
@@ -174,7 +174,7 @@ rules:
   func-call-spacing: [warn, never]
   function-paren-newline: [warn, consistent]
   implicit-arrow-linebreak: [warn, beside]
-  indent: [warn, 2, {CallExpression: {arguments: first}, SwitchCase: 1}]
+  indent: [warn, 2, {CallExpression: {arguments: first}, SwitchCase: 1, ignoredNodes: ["JSXElement", "ArrowFunctionExpression"]}]
```

Comment:
Does it just ignore these lines or does it enforce the indent style mentioned in the commit message?

Reply:
@mwiencek ^

Reply:
It ignores those nodes. `JSXElement` nodes are handled separately by the react/jsx-indent rule, so that's intended. Ignoring `ArrowFunctionExpression` means it won't find any indentation errors on those, but I believe that's better than it forcing us to make unnecessary changes.